### ISSUE TITLE
[BUG][SDW]ipc error for 0x60070000 when try to resume paused aplay after wake from S2idle

### ISSUE THREAD

**Issue by ClarexZhou** [thesofproject/linux#1306]
**Describe the bug**
Enter S2idle during aplay paused. Aplay can't be resumed after wake up from S2idle.

**To Reproduce**
1. echo s2idle > /sys/power/mem_sleep
2. Terminal 1: aplay -Dhw:0,0 -r48000 -c2 -fs16_le /dev/zero -vv -i,  press SPACE button to pause aplay
3. Terminal 2: sudo rtcwake -m mem -s 10

**frequency**
100%

**Actual result**
ipc error for 0x60070000: GLB_STREAM_MSG: TRIG_RELEASE

**Expected result**
Aplay should be resumed.

**Environment**
Firmware: e755494  Master
Kernel: d2b55e0 on integration/soundwire-latest
Topology: file: same with firmware branch, sof-cml-rt700.m4
platform: CML RVP with onboard ALC700 in SDW mode

<details>
 <summary>Dmesg</summary>

````
[  110.372583] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.372593] sof-audio-pci 0000:00:1f.3: ipc tx: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.372798] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.372805] sof-audio-pci 0000:00:1f.3: ipc tx: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.373001] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.373009] sof-audio-pci 0000:00:1f.3: ipc tx: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.373207] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.373214] sof-audio-pci 0000:00:1f.3: ipc tx: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.373465] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x50010000: GLB_COMP_MSG: SET_VALUE
[  110.373473] sof-audio-pci 0000:00:1f.3: ipc tx: 0x40020000: GLB_PM_MSG: CTX_RESTORE
[  110.373611] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x40020000: GLB_PM_MSG: CTX_RESTORE
[  110.373669] HDMI HDA Codec ehdaudio0D2: hdac_hdmi_present_sense: disconnect for pin:port 5:0
[  110.373673] HDMI HDA Codec ehdaudio0D2: hdac_hdmi_present_sense: disconnect for pin:port 6:0
[  110.373679] HDMI HDA Codec ehdaudio0D2: hdac_hdmi_present_sense: disconnect for pin:port 7:0
[  110.373805] rt700 sdw:1:25d:700:0:0: plb: sdw_modify_slave_status changing status slave 1 status 1 new status 0
[  110.373807] rt700 sdw:1:25d:700:0:0: plb: sdw_modify_slave_status initializing completion for Slave 1
[  110.375116] intel-sdw sdw-master-1: Slave status change
[  110.375121] intel-sdw sdw-master-1: sdw_handle_slave_status: start
[  110.375125] intel-sdw sdw-master-1: Slave attached, programming device number
[  110.375196] sof-audio-pci 0000:00:1f.3: ipc rx: 0x90020000: GLB_TRACE_MSG
[  110.375219] sof-audio-pci 0000:00:1f.3: ipc rx done: 0x90020000: GLB_TRACE_MSG
[  110.375322] intel-sdw sdw-master-1: SDW Slave Addr: 10025d070000
[  110.375324] intel-sdw sdw-master-1: SDW Slave class_id 0, part_id 700, mfg_id 25d, unique_id 0, version 1
[  110.375326] intel-sdw sdw-master-1: in sdw_assign_device_num
[  110.375327] intel-sdw sdw-master-1: in sdw_assign_device_num
[  110.375330] intel-sdw sdw-master-1: Slave already registered, reusing dev_num:1
[  110.375331] intel-sdw sdw-master-1: in sdw_assign_device_num, writing dev_num 1
[  110.376040] intel-sdw sdw-master-1: Msg Ack not received
[  110.376043] intel-sdw sdw-master-1: Msg Ack not received
[  110.376044] intel-sdw sdw-master-1: Msg Ack not received
[  110.376046] intel-sdw sdw-master-1: Msg Ack not received
[  110.376047] intel-sdw sdw-master-1: Msg Ack not received
[  110.376048] intel-sdw sdw-master-1: Msg Ack not received
[  110.376051] intel-sdw sdw-master-1: Msg ignored for Slave 0
[  110.376054] intel-sdw sdw-master-1: No more devices to enumerate
[  110.376056] intel-sdw sdw-master-1: End of sdw_program_device_num ret 0
[  110.376106] intel-sdw sdw-master-1: Slave status change
[  110.376112] intel-sdw sdw-master-1: sdw_handle_slave_status: start
[  110.376115] intel-sdw sdw-master-1: processing Slave 1 Status 1
[  110.376119] rt700 sdw:1:25d:700:0:0: plb: sdw_modify_slave_status changing status slave 1 status 0 new status 1
[  110.376121] rt700 sdw:1:25d:700:0:0: plb: sdw_modify_slave_status completing completion for Slave 1
[  110.376126] intel-sdw sdw-master-1: Slave 1 initializing
[  110.376296] intel-sdw sdw-master-1: Updating Slave 1 status
[  110.376299] intel-sdw sdw-master-1: sdw_handle_slave_status: Updating Slave 1 status done
[  110.376301] intel-sdw sdw-master-1: sdw_handle_slave_status: end
[  110.570476] OOM killer enabled.
[  110.570477] Restarting tasks ... done.
[  110.621640] PM: suspend exit
[  110.628350] atkbd serio0: Failed to enable keyboard on isa0060/serio0
[  110.875261] sof-audio-pci 0000:00:1f.3: ipc rx: 0x90020000: GLB_TRACE_MSG
[  110.875280] sof-audio-pci 0000:00:1f.3: ipc rx done: 0x90020000: GLB_TRACE_MSG
[  112.318138] asix 1-3.1:1.0 enx000ec6c686b2: link up, 100Mbps, full-duplex, lpa 0xCDE1
[  159.930376] sof-audio-pci 0000:00:1f.3: pcm: trigger stream 0 dir 0 cmd 4
[  159.931176] sof-audio-pci 0000:00:1f.3: FW Poll Status: reg=0x4001e successful
[  159.931177] sof-audio-pci 0000:00:1f.3: ipc tx: 0x60070000: GLB_STREAM_MSG: TRIG_RELEASE
[  159.931429] sof-audio-pci 0000:00:1f.3: error: ipc error for 0x60070000 size 12
[  159.931433]  SDW1-speakers: ASoC: trigger FE failed -22
[  159.931435] sof-audio-pci 0000:00:1f.3: pcm: trigger stream 0 dir 0 cmd 3
[  159.931437] sof-audio-pci 0000:00:1f.3: ipc tx: 0x60060000: GLB_STREAM_MSG: TRIG_PAUSE
[  159.931636] sof-audio-pci 0000:00:1f.3: error: ipc error for 0x60060000 size 12
[  159.932152] sof-audio-pci 0000:00:1f.3: FW Poll Status: reg=0x40002 timedout
[  159.932153]  SDW1-speakers: ASoC: trigger FE failed -22
[  160.875909] sof-audio-pci 0000:00:1f.3: ipc rx: 0x90020000: GLB_TRACE_MSG
[  160.875932] sof-audio-pci 0000:00:1f.3: ipc rx done: 0x90020000: GLB_TRACE_MSG
[  182.043291] sof-audio-pci 0000:00:1f.3: pcm: trigger stream 0 dir 0 cmd 4
[  182.043304] sof-audio-pci 0000:00:1f.3: FW Poll Status: reg=0x4001e successful
[  182.043315] sof-audio-pci 0000:00:1f.3: ipc tx: 0x60070000: GLB_STREAM_MSG: TRIG_RELEASE
[  182.043603] sof-audio-pci 0000:00:1f.3: error: ipc error for 0x60070000 size 12
[  182.043614]  SDW1-speakers: ASoC: trigger FE failed -22
[  182.043624] sof-audio-pci 0000:00:1f.3: pcm: trigger stream 0 dir 0 cmd 3
[  182.043634] sof-audio-pci 0000:00:1f.3: ipc tx: 0x60060000: GLB_STREAM_MSG: TRIG_PAUSE
[  182.043871] sof-audio-pci 0000:00:1f.3: error: ipc error for 0x60060000 size 12
[  182.044419] sof-audio-pci 0000:00:1f.3: FW Poll Status: reg=0x40002 timedout
[  182.044427]  SDW1-speakers: ASoC: trigger FE failed -22
````
</details>

[dmesg_s2idleFromAplayPaused.txt](https://github.com/thesofproject/linux/files/3715962/dmesg_s2idleFromAplayPaused.txt)

**Comment by ClarexZhou** [comment]
Will retest after #1196 fixed

**Comment by YvonneYang2** [comment]
Error can been seen on latest kernel and FW on icl 3-in-1 rvp

Environment
Firmware: c7d0149 https://github.com/thesofproject/sof/commits/master
Kernel: e7fb5a5 https://github.com/thesofproject/linux/tree/integration/soundwire-latest
tplg: sof-icl-rt711-rt1308-rt715-hdmi.tplg rename sof-icl-rt711-rt1308-rt715.tplg path same as FW branch
platform: ICL RVP, with codec ALC711 -RT1308, RT715 in SDW mode

**Comment by plbossart** [comment]
Please recheck with the latest integration/soundwire-latest (19df577a9096437d50ad77fa5af34f90a8f858da), these errors are gone for me.

The behavior is not fully right just yet, so added new issue https://github.com/thesofproject/linux/issues/1311

**Comment by YvonneYang2** [comment]
Test it with latest latest integration/soundwire-latest (19df577) on icl 3-in-1 rvp and cml rt700 rvp, these errors are gone. New behavior will track in #1311
